packages/client-react-streaming/src/DataTransportAbstraction/WrappedApolloClient.tsx

```diff
@@ -292,15 +280,21 @@ class ApolloClientSSRImpl extends ApolloClientClientBaseImpl {
   watchQuery<
     T = any,
```

Comment:
```suggestion
    TData = any,
```

Want to also do a quick rename on this to stick with `TData` as much as possible?